core/test/integration/synchronization/ripple_synchronization.cpp

```diff
@@ -108,6 +108,8 @@ TEST_F(RippleLikeWalletSynchronization, MediumXpubSynchronization) {
             auto block = wait(account->getLastBlock());
             auto blockHash = block.blockHash;
 
+            EXPECT_EQ(wait(account->isAddressActivated("rageXHB6Q4VbvvWdTzKANwjeCT4HXFCKX7")), true);
+            EXPECT_EQ(wait(account->isAddressActivated("rageXHB6Q4VbvvWdTzKANwjeCT4HXFCK")), false);
```

Comment:
This test only checks that address is valid, because it will fails the `parse` operation https://github.com/LedgerHQ/lib-ledger-core/pull/340/files#diff-212cd97f714ef22a58bc4f440cadb639R470
Could you please add the test when the address is valid but really not activated.

Reply:
I skipped that case because I didn't have any address that is valid but nor activated 😢

Reply:
Your wish is my command: `rf1pjatD8LyyevP1BqQJtHoz5edC5vE77Q `

Reply:
![](https://phaazon.net/media/uploads/good.jpg)

Reply:
Thanks @onyb ! But never activate it ! Otherwise you break our tests ! :trollface:

Reply:
one from XRP documentation - rDTXLQ7ZKZVKz33zJbHjgVShjsBnqMBhmN (with private key generation, so I hope nobody will ever activate it, and if you see the test failing, quickly call me)